Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -118,11 +147,24 @@ class ContentBlockerHelper {
         self.tab = tab
         self.profile = profile
 
+        if AppConstants.IsRunningTest {
+            ContentBlockerHelper.testInstance = self
+        }
+
         if ContentBlockerHelper.heavyInitHasRunOnce {
             return
         }
         ContentBlockerHelper.heavyInitHasRunOnce = true
 
+        // Read the whitelist at startup
+        if let dir = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first {
+            let fileURL = dir.appendingPathComponent("whitelist")
```

Comment:
Actually, we should maybe pull this bit of code out that gets the `whitelist` file into a private helper function on this class so we can re-use it in both places.

Reply:
will do